### PR TITLE
Fix the Audio Listener 3D property not being grouped with 2D in Viewport

### PR DESCRIPTION
**Pull request by Calinou**
This is a regression in `master`; the original issue is not present in `3.x`.

### CODE REVIEW

File: scene/main/viewport.cpp [hunk 1]
```diff
@@ -3754,6 +3753,7 @@ void Viewport::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "canvas_item_default_texture_repeat", PROPERTY_HINT_ENUM, "Disabled,Enabled,Mirror"), "set_default_canvas_item_texture_repeat", "get_default_canvas_item_texture_repeat");
 	ADD_GROUP("Audio Listener", "audio_listener_");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "audio_listener_enable_2d"), "set_as_audio_listener_2d", "is_audio_listener_2d");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "audio_listener_enable_3d"), "set_as_audio_listener_3d", "is_audio_listener_3d");
```
Comment (akien-mga) [inline scene/main/viewport.cpp:3756]: Did you check that it's actually defined when 3D is disabled? That's likely why it had been moved in a `!_3D_DISABLED` block.

Comment (Calinou) [reply scene/main/viewport.cpp:3757]: I added a `#ifndef _3D_DISABLED` check.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by aaronfranke** [comment]
I'm pretty sure this was my fault, sorry about that!

**Pull request merged**